### PR TITLE
Created Parser Plugin for validation errors   issue#675

### PR DESCRIPTION
**Pull request by jaynarayan89**

### CODE REVIEW

File: system/View/Plugins.php [hunk 1]
```diff
@@ -121,5 +121,26 @@ public static function lang(array $params = [])
...
+	 */
+	public static function ValidationErrors(array $params = [])
+	{
+		
+		$validator = \config\services::validation();
+		if(empty($params))
+		{
+			return $validator->listErrors();
+		}
+		else
```
Comment (samsonasik) [inline system/View/Plugins.php]: no need else as already return early

Comment (lonnieezell) [reply system/View/Plugins.php]: Agreed. Please remove the else here.

**Comment by jaynarayan89** [comment]
Ok. I will make changes.
Another thing I want to point out that we should have plugin for viewcell and form helper methods as well.
If you agree I would happy to help.

**Comment by jim-parry** [comment]
@jaynarayan89  Just pointing out that we would like to see contributions GPG-signed ... https://bcit-ci.github.io/CodeIgniter4/contributing/signing.html

**Comment by jaynarayan89** [comment]
Hi , I accidentally merged another branch with this and messed up commits. I tried to revert but not succeed. what should I do now?  
I am learning github. sorry for the trouble.

@jim-parry ok. I will do it.

**Comment by jim-parry** [comment]
@jaynarayan89 Your last "good" (pre patch2) merge shows a commit hash tag starting with "17fa3d4". Assuming that is the "good" point you want, you should be able to revert to that with "git reset --hard 17fa3d4...", and then "git push -f origin patch-2" to fix your feature branch.

**Comment by jaynarayan89** [comment]
Thanks a lot @jim-parry 
I need another help regarding Unit testing. when i run phpunit in my root folder it shows:

PHPUnit 3.7.21 by Sebastian Bergmann.
Configuration read from D:\xampp\htdocs\ci4.dev\CodeIgniter4\phpunit.xml
Time: 921 ms, Memory: 10.00MB

←[30;43m←[2KNo tests executed!
←[0m←[2K
Generating code coverage report in Clover XML format ... done

**Comment by jim-parry** [comment]
@jaynarayan89 I run unit tests with the following command from my project root...
phpunit --exclude-group DatabaseLive --colors --coverage-text=tests/coverage.txt --coverage-html=tests/coverage/

**Comment by jaynarayan89** [comment]
@jim-parry 
Again same messages.
D:\xampp\htdocs\ci4.dev\CodeIgniter4>phpunit --exclude-group DatabaseLive --colo
rs --coverage-text=tests/coverage.txt --coverage-html=tests/coverage/
PHPUnit 3.7.21 by Sebastian Bergmann.

Configuration read from D:\xampp\htdocs\ci4.dev\CodeIgniter4\phpunit.xml
Time: 505 ms, Memory: 10.00MB

←[30;43m←[2KNo tests executed!
←[0m←[2K
Generating code coverage report in Clover XML format ... done
Generating code coverage report in HTML format ... done

**Comment by jaynarayan89** [comment]
@jim-parry 
Regarding phpunit version:
I installed phpunit  via composer an hour ago. using this command
composer require --dev phpunit/phpunit ^6.2
however it installed old version. I tried but cant install latest version.

**Comment by jim-parry** [comment]
@jaynarayan89 Rather than clutter up this commit history, can you start a thread with me on our slack channel? thanks ... http://codeignitersignup.ciblox.com/

**Comment by jaynarayan89** [comment]
@lonnieezell 
Please review changes

**Comment by lonnieezell** [comment]
I will, but you have to give me more than an hour. :) It's the middle of my work day here, and, unfortunately, I don't make a living developing this so won't be until tonight (hopefully!) before I can check it. Though the way the last couple of weeks have been going, I'll probably be working really late.

**Pull request merged**